### ISSUE TITLE
F12

### ISSUE THREAD

**Issue by idoo** [otoniel-isidoro/vscode-ruby-ctags#1]
Thank you for this package.
Are here some options to integrate `go to definitions` to F12 (default key for this command in VSC)

**Comment by otoniel-isidoro** [comment]
Yes. I alredy put it on my TODO list on README.md. But I am currently out of time. 

Here is the TODO list:

> Integration with VSCODE Go To Definition vscode-api#DefinitionProvider.provideDefinition
> Integration with VSCODE Go to Symbol vscode-api#languages.registerDocumentSymbolProvider
> Go To Definition with mouse click
